src/hashsearch.py

```diff
@@ -76,28 +81,40 @@ def main(**kwargs):
     config["offline"] = kwargs["offline"]
     config["wordlist"] = kwargs["wordlist"]
     config["hashcat"] = kwargs["hashcat"]
+    config["timeout"] = kwargs["timeout"]
+    config["greppable"] = kwargs["greppable"]
+
+    Prettifier.banner()
+
+    searcher = cracking.Searcher(config)
+    cracking.Searcher.main(searcher)
 
-    config = searcher.Searcher(config)
+    # printing.Prettifier(results, config)
+
+
+def return_as_json(hashes):
+
+    config = {}
+
+    config["hashes"] = hashes
+    config["offline"] = False
+    config["timeout"] = 1
+    config["greppable"] = True
+
+    searcher = cracking.Searcher(config)
+
+    return cracking.Searcher.main(searcher)
 
 
 def create_hash_config(config):
-    # Returns the hashing config
-    # [{hash: {hash_types}}, {hash: {hash_types}}]
-    result = []
-    for hash in config["hashes"]:
-        try:
-            result.append({hash: get_hashid(hash)})
-        except Exception as e:
-            print(f"{hash} is an unknown hash type.")
-            continue
-    return result
+    try:
+        return json.loads(runner.api_return_hashes_as_json(config["hashes"]))
+    except:
+        print("Invalid hash type")
+        exit(0)
 
 
-def get_hashid(hash):
-    hash_identifier = hashid.HashID()
-    return hashid.writeResult(
-        hash_identifier.identifyHash(hash)  # , None, True, True, True
-    )
+# CONFIG FILE FUNCTIONS VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
 
 
 def read_config_file():
```

Comment:
is this functiion empty?

Reply:
you need to call this in main, update the config from the args with the config file

Reply:
and test it!